vagen/envs/tos/managers/cognitive_map_manager.py

```
@@ -0,0 +1,1516 @@
+                cogmap_str,
+                room,
+                agent,
+                list(observed_items),
+                map_type="global",
+            )
+            if turn_log and turn_log.metrics.valid:
+                pos = float(turn_log.metrics.pos) if turn_log.metrics.pos is not None else 0.0
+                facing = float(turn_log.metrics.facing) if turn_log.metrics.facing is not None else 0.0
+                return max(0.0, min(1.0, (pos + facing) / 2.0))
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)